### PR TITLE
[DOCS] Input

### PR DESCRIPTION
**Pull request by mhilbrunner**

### CODE REVIEW

File: doc/classes/Input.xml [hunk 1]
```diff
@@ -44,7 +45,7 @@
 			<return type="Vector3">
 			</return>
 			<description>
-				If the device has an accelerometer, this will return the movement.
+				If the device has an accelerometer, this will return the acceleration.
```
Comment (LikeLakers2) [inline doc/classes/Input.xml]: Just gonna ask this for completions sake: What if it doesn't have an accelerometer, or doesn't have one that Godot can access? What will it return then?

Comment (Noshyaar) [reply doc/classes/Input.xml]: I think it returns `Vector3()`.

**Comment by mhilbrunner** [comment]
@LikeLakers2 @Noshyaar Updated, get_accelerometer(), get_gravity() and get_gyroscope() indeed return empty Vector3()s when called on deviced without such sensors.

**Comment by hpvb** [comment]
Cherry picked into 3.0.3

**Pull request merged**